docs/userguide/advanced/monitoring.rst

```diff
@@ -117,3 +170,35 @@ The workflow summary also presents three different views of the workflow:
 
 .. image:: ../../images/mon_resource_summary.png
 
+* Workflow file provenance (only if enabled and files were used in the workflow): This visualization gives a tabular listing of each task that created (output) or used (input) a file. Each listed file has a link to a page detailing the file's information.
```

Comment:
You can remove the caveats in the parentheses. This section is purely a description of the figures.